assignment/assignment-tool/tool/src/webapp/js/assignments.js

```diff
@@ -726,10 +726,10 @@ ASN.invokeDownloadUrl = function(accessPointUrl, actionString, alertMessage, par
     if (document.getElementById('feedbackAttachments') && document.getElementById('feedbackAttachments').checked)
     {
         extraInfoArray[extraInfoArray.length]="feedbackAttachments=true";
-    }

```

Comment:
It doesn't look like this block has changed.

Reply:
LF changed to CRLF, not sure why it is just for this block.  Can you fix pls?

Reply:
Done.